src/uwtools/tests/config/formats/test_base.py

```diff
@@ -159,8 +159,8 @@ def test_update_values(config):
     assert config == {"foo": 88, "a": "11", "b": "12", "c": "13"}
 
 
-@pytest.mark.parametrize("fmt1", [FORMAT.ini, FORMAT.nml, FORMAT.yaml])
-@pytest.mark.parametrize("fmt2", [FORMAT.ini, FORMAT.nml, FORMAT.yaml])
+@pytest.mark.parametrize("fmt1", [FORMAT.ini, FORMAT.nml, FORMAT.sh, FORMAT.yaml])
```

Comment:
These tests are currently failing due to the single-level of bash files. This was missed before and will need to be handled now. Still working on how I want to do that, however.

Reply:
I think the above suggestion of defining `MAXDEPTH` will let me code the writes better! Work in progress

Reply:
It could be a module- or class-level constant, or a `@property` method. I think the latter two are probably more appropriate so that, if you've got a reference to a `Config` instance, you can get its maximum depth without referring back the module where the instance is defined. Between class constant and `@property` method, its probably a matter of taste.